scripts/install/opt/seagate/cortx/motr/libexec/prov-m0-reset

```diff
@@ -82,6 +87,8 @@ while true; do
         --start-consul)      start_consul_only=true; shift ;;
         --stop-consul)       stop_consul_only=true; shift  ;;
         --mkfs-only)         mkfs_only=true; shift ;;
+        --single-node)       single_node=true; shift 2 ;;
```

Comment:
--single-node=true or --single-node=false?

This is a must?
"--single-node" can be used with the second following argument?

Reply:
This single-node option is used just like mkfs-only option. If provided on command-line only then we assume it to the true.

Reply:
Because the "--single-node" is the last argument in current script, it does not fail.
But in future, if it is in the middle of a list of arguments, this will skip the next valid argument.